falco/values-k8saudit.yaml

```diff
@@ -1,11 +1,34 @@
+# -- Disable the drivers since we want to deplouy only the k8saudit plugin.
 driver:
   enabled: false
 
+# -- Disable the collectors, no syscall events to enrich with metadata.
 collectors:
   enabled: false
 
+# -- Deploy Falco as a deployment. One instance of Falco is enough. Anyway the number of replicas is configurabale.
 controller:
   kind: deployment
+  deployment:
+    # -- Number of replicas when installing Falco using a deployment. Change it if you really know what you are doing.
+    # For more info check the section on Plugins in the README.md file.
+    replicas: 1
+
+
+falcoctl:
+  artifact:
+    install:
+      # -- Enable the falcoctl tool as init container. It installs artifacts in the config.artifact.install.refs list.
+      enabled: true
+    follow:
+      # -- Disable the sidecar container. We do not support it yet for plugins. It is used only for rules feed such as Falco rules.
+      enabled: false
+  config:
+    artifact:
+      install:
+        # -- List of artifacts to be installed by the falcoctl init container.
+        # Same plugins we are loading in Falco. See "load_plugins" section.
+        refs: [k8saudit:0, json:0]
```

Comment:
```suggestion
falcoctl:
  artifact:
    install:
      # -- Disable the init container. We do not recommend installing (or following) plugins for security reasons since they are executable objects.
      enabled: false
    follow:
      # -- Enable the sidecar container. We do not support it yet for plugins. It is used only for rules feed such as k8saudit-rules rules.
      enabled: true
  config:
    artifact:
      follow:
        # -- List of artifacts to be followed by the falcoctl sidecar.
        # Assuming plugins k8saudit v0.5.0 and json v0.6.0 shipped in-bundle with Falco 0.34
        refs: [k8saudit-rules:0.5]
```